### ISSUE TITLE
Plugin loading fails to encoding(?) issue on AppVeyor

### ISSUE THREAD

**Issue by perrenyang** [CCSI-Toolset/FOQUS#462]
I've been trying to get FOQUS installed/built and running on AppVeyor for automated testing on a Windows platform. When the application attempts to launch, an error occurs during plugin loading:

```
2019-04-01 19:46:43,144 - DEBUG - foqus.foqus_lib.framework.session.session - Initializing session, Log file: C:\projects\tests\logs\foqus.log, Position: 0
2019-04-01 19:46:43,144 - INFO - foqus.foqus_lib.framework.plugins.pluginSearch - Loading Plugin: C:\projects\foqus-37w98\foqus_lib\framework\surrogate\ACOSSO.py
2019-04-01 19:46:43,144 - CRITICAL - foqus.__main__ - unhandled exception
Traceback (most recent call last):
  File "foqus.py", line 398, in <module>
    dat = session(useCurrentWorkingDir=True)
  File "C:\projects\foqus-37w98\foqus_lib\framework\session\session.py", line 239, in __init__
    self.loadPlugins()
  File "C:\projects\foqus-37w98\foqus_lib\framework\session\session.py", line 343, in loadPlugins
    charLimit = 150)
  File "C:\projects\foqus-37w98\foqus_lib\framework\plugins\pluginSearch.py", line 28, in __init__
    self.importPlugins()
  File "C:\projects\foqus-37w98\foqus_lib\framework\plugins\pluginSearch.py", line 42, in importPlugins
    l = self.idString in f.read(self.charLimit)
  File "C:\Miniconda3\envs\foqus\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 2835: character maps to <undefined>
Command exited with code 1
```
More complete context can be seen in the image below:
![image (1)](https://user-images.githubusercontent.com/44816826/55441605-38d1b680-5560-11e9-9119-776a81713427.png)

For comparison, when I run the same test on my own Windows machine, I see the GUI pop open and move through several different actions before exiting, leaving the following output:

![image](https://user-images.githubusercontent.com/44816826/55441668-720a2680-5560-11e9-9683-5939b4c5f255.png)

(probably you'll need to click on the images to load the full resolution)

**Comment by eslickj** [comment]
I've seen this too.  For me it happened the first time I tried to open FOQUS before the contents of the working directory where created.  The second time I tried to run it worked fine.  I'll look into it.

**Comment by eslickj** [comment]
@fsoepyan also reported this in issue #463, which I'm going to close as a duplicate.  I'm not sure why this started happening, but a least for me it works the second time.

**Comment by eslickj** [comment]
Should be fixed in PR #464

**Comment by eslickj** [comment]
I think it's all succeeding now.
